.npmignore

```diff
@@ -0,0 +1,9 @@
+*.DS_Store
```

Comment:
If this file is present, it should be used to filter files during packaging instead of the .gitignore